src/containers/org/components/RemoveMemberFromOrgModal.js

```diff
@@ -12,15 +12,18 @@ import { RequestStates } from 'redux-reqseq';
 import type { UUID } from 'lattice';
 import type { RequestState } from 'redux-reqseq';
 
+import ResetOnUnmount from './ResetOnUnmount';
```

Comment:
Why not use the one in `components/other`?

Reply:
Modals that are still using the ActionModal will all have to change. And I suspect they'll all have to be refactored in the near future to make way for Snackbar confirmations instead. Enhancement seems to be out of the scope of this PR